.spectral.yaml

```diff
@@ -2,18 +2,125 @@ formats:
+    severity: error
     given: $.info
-    recommended: true
-    type: style
     then:
       field: title
       function: pattern
       functionOptions:
         match: Whispir
-  openapi-tags-alphabetical: off
-
+  schema-examples:
+    description: Schemas must have non-empty "x-examples" object.
+    severity: error
+    given: $.components.schemas.*
+    then:
+      field: x-examples
+      function: schema
+      functionOptions:
+        dialect: draft7
+        schema:
+          type: object
+          minProperties: 1
+  schema-descriptions:
+    description: Schemas must have a description.
+    severity: error
+    given: $.components.schemas.*
+    then:
+      field: description
+      function: truthy
+  schema-property-descriptions:
+    description: Schema properties must have a description.
+    severity: error
+    given: $.components.schemas.*..properties.*
+    then:
+      field: description
+      function: truthy
+  schema-property-examples:
+    description: Schema properties must have a example. This includes $ref'd schemas.
+    severity: error
+    given: $.components.schemas.*..properties.*
+    then:
+      field: example
+      function: truthy
+overrides:
+  - rules:
+      schema-examples: off
+      schema-descriptions: off
+      schema-property-descriptions: off
+      schema-property-examples: off
+    files:
+      - '**#/components/schemas/message' # Message Endpoint Schemas
+      - '**#/components/schemas/email'
+      - '**#/components/schemas/voice'
+      - '**#/components/schemas/web'
+      - '**#/components/schemas/social'
+      - '**#/components/schemas/resource'
+      - '**#/components/schemas/template'
+      - '**#/components/schemas/features'
+      - '**#/components/schemas/pushOptions'
+      - '**#/components/schemas/attachment'
+      - '**#/components/schemas/responseTemplatePattern' # Response Template Endpoint Schemas
+      - '**#/components/schemas/responseRule'
+      - '**#/components/schemas/contact' # Contact Endpoint Schemas
+      - '**#/components/schemas/messagingOptions'
+      - '**#/components/schemas/messagingOption'
+      - '**#/components/schemas/distributionlist' # Distribution List Endpoint Schemas
+      - '**#/components/schemas/locations'
+      - '**#/components/schemas/location'
+      - '**#/components/schemas/scenario' # Scenario Endpoint Schemas
+      - '**#/components/schemas/callback' # Callback Endpoint Schemas
+      - '**#/components/schemas/callbacks'
+      - '**#/components/schemas/callbackParameters'
+      - '**#/components/schemas/workspace' # Workspace Endpoint Schemas
+      - '**#/components/schemas/user' # User Endpoint Schemas
+      - '**#/components/schemas/event' # Event Endpoint Schemas
+      - '**#/components/schemas/import' # Import Endpoint Schemas
+      - '**#/components/schemas/fieldMapping'
+      - '**#/components/schemas/auth' # Auth Endpoint Schemas
+      - '**#/components/schemas/activity' # Activity Endpoint Schemas
```

Comment:
We'll iteratively remove these overrides as we address gap-fills in all endpoints.